src/styles.scss

```diff
@@ -2,7 +2,7 @@
 
 // Banner button size, can be changed
 $bannerBtnWidth: 150px;
-$bannerBtnHeight: 36px;
+$bannerBtnHeight: 45px;
```

Comment:
Now buttons look too big for regular content.
Can you try to make the buttons size flexible, depending on the content? With some min-width and min-height in place.